docs/intro/tutorial04.md

```diff
@@ -0,0 +1,88 @@
+# Writing your first app, part 4
+
+This tutorial begins where [Tutorial 3](tutorial03.md) left off. We’ll walk through the process for installing your
+first application written in WebAssembly into your Kubernetes cluster, then test our application using `kubectl`.
+
+## Scheulding pods on the Krustlet
+
+In Kubernetes, Pods are the smallest deployable units of compute that can be created and managed in Kubernetes. In other
+words, your application runs inside a Pod, and we can inspect the status of the application by inspecting the Pod.
+
+Krustlet listens for pods requesting a node with the `wasm32-wasi` architecture. To schedule a Pod that Krustlet
+understands, we need to provide Kubernetes with a YAML file describing our Pod.
+
+Create a new file and call it `krustlet-demo.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: krustlet-demo
+spec:
+  containers:
+    - name: krustlet-demo
+      image: mycontainerregistry007.azurecr.io/krustlet-tutorial:v1.0.0
+  nodeSelector:
+    beta.kubernetes.io/arch: wasm32-wasi
+```
+
+Let's break this file down:
+
+- `apiVersion`: which version of the Kubernetes API are we targeting?
+- `kind`: what type of workload are we deploying?
+- `metadata.name`: what is the name of our workload?
+- `spec.containers[0].name`: what should I name this module?
+- `spec.containers[0].image`: where can I find the module?
+- `spec.nodeSelector.beta.kubernetes.io/arch`: what chip architecture can run this workload?
+
+To deploy this workload to Kubernetes, we use `kubectl`.
+
+```console
+$ kubectl create -f krustlet-demo.yaml
+```
+
+Now that the workload has been scheduled, Krustlet should start spewing out some logs in its terminal window, reporting
```

Comment:
So _technically_ krustlet can't do this yet. I assume this is supposed to be forward thinking for when it does, correct?

Reply:
Yep, wishful thinking

Reply:
The rest of part 4 is all wishful thinking, too. I haven't actually been able to get something working, but I wrote this with the intention that *this is the happy path we are striving for*. :smile: